app/services/default_form_builder.rb

```diff
@@ -0,0 +1,32 @@
+class DefaultFormBuilder
+  FIELDS = [
```

Comment:
I like this service but I'm not very happy with the settings for the default form buried in the codebase... to me it seems like this should be in a config file. I would say even on the `Form` model definition would be easier to find. what do you think?